lms/djangoapps/ccx/views.py

```diff
@@ -326,6 +327,13 @@ def override_fields(parent, data, graded, earliest=None, ccx_ids_to_delete=None)
     if changed:
         override_field_for_ccx(ccx, course, 'grading_policy', policy)
 
+    responses = SignalHandler.course_published.send(
+        sender=ccx,
```

Comment:
If I am not wrong, the sender is usually the generic class that is sending the signal, not the specific object: not sure this matters here.